templates/docs/examples/patterns/notifications/toast.html

```diff
@@ -11,26 +11,51 @@
 {% set timestamp = request.args.get('timestamp') %}
 
 {% block content %}
-<div class="p-notification{{ '--' ~ type if type else '' }} {% if theme == 'dark' %}is-dark{% endif %} {% if style == 'inline' %}is-inline{% endif %}">
+<div class="p-notification{{ '--' ~ type if type else '' }}{% if theme == 'dark' %} is-dark{% endif %}{% if style == 'inline' %} is-inline{% endif %}">
   <div class="p-notification__content">
     <h5 class="p-notification__title">Permissions changed</h5>
     <p class="p-notification__message">Anyone with access can view your invited users.</p>
-    {% if dismiss == 'true' %}
+    {%- if dismiss == 'true' %}
     <button class="p-notification__close" aria-controls="notification">Close</button>
-    {% endif %}
+    {%- endif %}
   </div>
-  {% if actions == 'true' or timestamp == 'true' %}
+  {%- if actions == 'true' or timestamp == 'true' %}
   <div class="p-notification__meta">
-    {% if timestamp == 'true' %}
+    {%- if timestamp == 'true' %}
     <span class="p-notification__timestamp">1h ago</span>
-    {% endif %}
-    {% if actions == 'true' %}
+    {%- endif %}
+    {%- if actions == 'true' %}
     <div class="p-notification__actions">
       <a class="p-notification__action" href="#">Action 1</a>
       <a class="p-notification__action" href="#">Action 2</a>
     </div>
-    {% endif %}
+    {%- endif %}
   </div>
-  {% endif %}
+  {%- endif %}
 </div>
+{%- if dismiss == 'true' %}
+<script>
+/**
+  Attaches event listener for hide notification on close button click.
+  @param {HTMLElement} closeButton The notification close button element.
+*/
+function setupCloseButton(closeButton) {
+  closeButton.addEventListener('click', function(event) {
+    var target = event.target.getAttribute('aria-controls');
+    var notification = document.getElementById(target);
+
+    if (notification) {
+      notification.classList.add('u-hide');
+    }
+  });
+}
+
+// Set up all notification close buttons.
+var closeButtons = document.querySelectorAll('.p-notification__close');
+
+for (var i = 0, l = closeButtons.length; i < l; i++) {
+  setupCloseButton(closeButtons[i]);
+}
+</script>
```

Comment:
This script doesn't seem to be working.

Reply:
@bethcollins92 asked @bartaz about this functionality and it seems to be best to remove this functionality for now! :)

Reply:
It's working now, I'd just forgotten to add `id="notification"` on the div :)